src/web/src/shared/use-cases/validate-ssp-xml.ts

```diff
@@ -74,7 +74,7 @@ const generateSchematronReport = (
 const detectFormat = (document: string) => {
   // Naive detection of JSON format - first non-whitespace character should be
   // `{` or `[`.
-  if (/\s*[{\[]/.test(document)) {
+  if (/^\s*[\{\[]/.test(document)) {
```

Comment:
So the only practical detection method is to look for `{` and `}` to signify JSON data?

Reply:
We could just do `JSON.parse(document)` in a `try... catch` block, but this avoids the cpu time and is really simple.